decidim-core/app/views/decidim/shared/_redesigned_login_modal.html.erb

```diff
@@ -0,0 +1,55 @@
+<%= decidim_modal id: "loginModal", class: "login__modal" do %>
```

Comment:
_decidim / staging123_

From the last weekend on, we put our servers behind authentication to avoid spam. I'll update the open PRs with that credentials